eng/common/native/init-distro-rid.sh

```diff
@@ -20,7 +20,7 @@ getNonPortableDistroRid()
         # shellcheck disable=SC1091
         if [ -e "${rootfsDir}/etc/os-release" ]; then
             . "${rootfsDir}/etc/os-release"
-            if [ "${ID}" = "rhel" ] || [ "${ID}" = "rocky" ] || [ "${ID}" = "alpine" ]; then
+            if [ "${ID}" = "rhel" ] || [ "${ID}" = "rocky" ] || [ "${ID}" = "alpine" || "${ID}" == "ol" ]; then
```

Comment:
s/` || `/`] || [`/ ? (should match the others)

Reply:
(and single `=`)

Reply:
```suggestion
            if [ "${ID}" = "rhel" ] || [ "${ID}" = "rocky" ] || [ "${ID}" = "alpine" ] || [ "${ID}" == "ol" ]; then
```